noxfile.py

```diff
@@ -156,16 +155,15 @@ def docs(session, pandas_version: str):
 # to automate the action workflow, leveraging its ability to set up the environment
 # required for model autotracing.
 @nox.session(python=["3.9"])
-@nox.parametrize("pandas_version", ["1.5.0"])
-def trace(session, pandas_version: str):
+def trace(session):
     session.install(
         "-r",
         "requirements-dev.txt",
         "--timeout",
         "1500",
     )
     session.install(".")
-    session.run("python", "-m", "pip", "install", f"pandas~={pandas_version}")
+    session.run("python", "-m", "pip", "install", "-r", "requirements.txt")
```

Comment:
Do we need this?   we are already installing `requirements-dev.txt` in the previous line.